services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/service/ovs/DataPlaneServiceImpl.java

```diff
@@ -557,12 +557,11 @@ private UnicastGoalState buildUnicastGoalState(NetworkConfiguration networkConfi
         multicastGoalState.setGoalState(multicastGoalState.getGoalStateBuilder().build());
         multicastGoalState.setGoalStateBuilder(null);
 
-        // TODO: Find a field to decide client
-        if (true){
-            return pulsarDataPlaneClient.createGoalStates(unicastGoalStates, multicastGoalState);
+        if (networkConfig.getPortEntities().get(0).isFastPath()){
```

Comment:
I have a question about this. Will there be different isFastPath values in multiple portEntity? How to deal with this situation.

Reply:
IsFastPath is a port-level config. Although we allow different values for multiple port entities for the sake of flexibility, we don't have such a scenario for now. In terms of implementation, we could make a port-level decision.

Reply:
In this case, for the same NetworkConfiguration, we need to divide it into two groups to generate Goalstate based on whether it is fastpath. Am I following this logic to modify

Reply:
If a NetworkConfiguration includes multiple port entities, DPM will process each entity independently and likely send the corresponding GS to its target host. 

At the moment of sending the GS, we need to use the isFastPath field to determine go with the fast path or MQ scaling path.